sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -217,6 +222,7 @@ export default defineComponent({
         isFulltimeAllowed,
       };
       existingApplication.value = applicationData;
+      savedDraftData = structuredClone(initialData.value);
```

Comment:
The first initialization can be done on page change event. Please keep in mind the first ever page has no input in it and any change to form can possibly happen only beyond a first page change.

![image](https://github.com/user-attachments/assets/81a57abf-7bfa-402d-b78b-8f7d74bf2635)